numpy/lib/__init__.py

```diff
@@ -21,6 +21,24 @@
 from . import scimath as emath
 
 # Private submodules
+# load module names. See https://github.com/networkx/networkx/issues/5838
+from . import type_check
+from . import index_tricks
+from . import function_base
+from . import nanfunctions
+from . import shape_base
+from . import stride_tricks
+from . import twodim_base
+from . import ufunclike
+from . import histograms
+from . import polynomial
+from . import utils
+from . import arraysetops
+from . import npyio
+from . import arrayterator
+from . import arraypad
+from . import _version
```

Comment:
There aren't really harmful imports to add, because they're already public-ish today; we should clean this up later though. Again no one should be using these namespaces directly; I wonder if the `LazyLoader` issue can also be fixed by not using the fully-qualified names elsewhere?